src/service/ActivityReportService/ActivityReportV2Controller.ts

```diff
@@ -43,6 +43,24 @@ export class ActivityReportV2Controller extends APIClient {
         });
     }
 
+    /**
+     * Prepares the body of the activity report email with all placeholders already filled by Lambda service.
+     * @param links array of links to activity reports
+     * @param estateId entity id of the estate object
+     */
+    async fetchEmailContent(links: ActivityReportLinkType[], estateId: string): Promise<ApiResponse<LambdaServiceResponse>> {
```

Comment:
You do not need a response here, because it will be automatically interpreted by the return statement

```suggestion
    async fetchEmailContent(links: ActivityReportLinkType[], estateId: string) {
```

Reply:
And maybe it makes more sense the switch the arguments? So have the `estateId` as first and then the `links`, because its an array ?